CONTRIBUTING.md

```diff
@@ -206,10 +206,20 @@ Please follow these steps to have your contribution considered by the approvers:
 
 1. Ensure all commits have a Sign-off for DCO, as described in [DCO.md].
 2. Follow all instructions in [PULL-REQUEST-TEMPLATE.md](.github/pull_request_template.md).
-3. Include appropriate test coverage. Testing is 100% automated. There is no such thing as a manual test.
+3. Include appropriate test coverage. Testing is 100% automated. All submissions must be testable in an automated fashion.
 4. Follow the [Style Guides](#style-guides).
 5. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) 
 are passing.
+
+#### What Makes A Good Pull Request? 
+
+The following guidelines, based on Hyperledger Fabic's [contribution guidelines](https://hyperledger-fabric.readthedocs.io/en/latest/CONTRIBUTING.html#what-makes-a-good-pull-request) will help ensure that your pull request gets promptly reviewed. 
+
+<details><summary>One Pull Request, One Change</summary>This limits the surface area of the change, and makes it easier to identify root causes when issues arise.</details>
+<details><summary>Link to JIRA</summary>When submitting your PR, include the JIRA ticket's link in the description and number in the title (i.e. `[BESU-99] My Awesome PR`), this helps provide more context on your work and auto-update the JIRA ticket to include a link to your PR.</details>
```

Comment:
@NicolasMassart do you know why the backticks around `[BESU-99] My Awesome PR` and `[BESU-99]` on Line 221 aren't working, while they are in the rest of the file?